jsapp/js/account/add-ons/addOnList.component.tsx

```diff
@@ -60,118 +60,155 @@ const AddOnList = (props: {
-                )}
-              </td>
-              <td className={styles.price}>{price.human_readable_price}</td>
-              <td>
-                {isSubscribedAddOnPrice(price) && (
-                  <BillingButton
-                    size={'m'}
-                    label={t('Manage')}
-                    isDisabled={props.isBusy}
-                    onClick={onClickManage}
-                    isFullWidth
-                  />
-                )}
-                {!isSubscribedAddOnPrice(price) && (
-                  <BillingButton
-                    size={'m'}
-                    label={t('Buy now')}
-                    isDisabled={props.isBusy}
-                    onClick={() => props.onClickBuy(price)}
-                    isFullWidth
-                  />
-                )}
-              </td>
-            </tr>
-          ));
+          return null;
         })}
       </tbody>
     </table>
   );
+
+  return (
+    <>
+      <table className={styles.table}>
+        <caption className={styles.caption}>
+          <label className={styles.header}>{t('available add-ons')}</label>
+          <p>
+            {t(
+              `Add-ons can be added to your Community plan to increase your usage limits. If you are approaching or
+              have reached the usage limits included with your plan, increase your limits with add-ons to continue
+              data collection.`
+            )}
+          </p>
+        </caption>
+        <tbody>
+          {props.organization && (
+            <>
+              {!subscribedPlans.length && (
+                <OneTimeAddOnRow
+                  key={filteredAddOnProducts
+                    .map((product) => product.id)
+                    .join('-')}
+                  products={filteredAddOnProducts}
+                  isDisabled={props.isBusy}
+                  organization={props.organization}
+                />
+              )}
+              <OneTimeAddOnRow
+                key={oneTimeAddOneProducts
+                  .map((product) => product.id)
+                  .join('-')}
+                products={oneTimeAddOneProducts}
+                isDisabled={props.isBusy}
+                organization={props.organization}
+              />
+            </>
+          )}
+        </tbody>
+      </table>
+      {subscribedAddOns.some((product) => product.status === 'active') ||
+      oneTimeAddOnSubscriptions.some(
+        (oneTimeAddOns) => oneTimeAddOns.is_available
+      )
+        ? renderActivePreviousAddons(
+            subscribedAddOns,
+            oneTimeAddOnSubscriptions,
+            'active',
+            true,
+            t('your active add-ons'),
+            'Active',
+            'light-teal'
+          )
+        : null}
+
+      {subscribedAddOns.some((product) => product.status !== 'active') ||
+      oneTimeAddOnSubscriptions.some(
+        (oneTimeAddOns) => !oneTimeAddOns.is_available
+      )
+        ? renderActivePreviousAddons(
```

Comment:
`renderActivePreviousAddons` should probably get renamed to `ActivePreviousAddons` and invoked as a React component here - it's a function that takes some props and returns JSX (or `null`), so it's a functional component.